proto/serverless/instrumentation/v1/trace.proto

```diff
@@ -74,4 +74,9 @@ message Span {
     // The optional custom tags to be set by the user
     // This is expected to be a JSON object in string format.
     optional string custom_tags = 13;
+
+    // Optional list of event fingerprints related to the span.
+    // This will most likely always be populated by ingest when looking at the events
+    // attached to the TracePayload
+    repeated string fingerprints = 14;
```

Comment:
Just so I am clear on what the plan is does this sound correct 👇


If we receive a trace payload with a set of custom events then ingest will attach the finger prints from each custom event to the root span (or any other span in the list)? Or will we only add specific fingerprints to specific spans?

Reply:
For the immediate term it will be all fingerprints to the root span. Longer term what we will probably do is attach all fingerprints to the root span and then specific fingerprints to the specific span in which the error occured.

Reply:
Ahh okay sounds good 👍

Reply:
@Mmarzex do we need to mark this as optional? That might be why the unit tests are failing

Reply:
@Danwakeem Repeated fields are inherently optional, in the sense that not setting them will encode/decode to an empty array, the tests are failing because of how `ts-proto` works. See [issue](https://github.com/stephenh/ts-proto/issues/225) on their repo. For now we will just do the required updates when we add fields like this. This is also as far as I can see only an issue for the node SDK, it will not be a problem for Python or go.

Reply:
Ahh okay sad times 😢